src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -207,7 +207,8 @@ Copyright (c) .NET Foundation. All rights reserved.
           FileDependencies="@(FileDependencies)"
           References="@(Reference)"
           DefaultImplicitPackages="$(DefaultImplicitPackages)"
-          InputDiagnosticMessages="@(DiagnosticMessages)">
+          InputDiagnosticMessages="@(DiagnosticMessages)"
+          TargetFrameworkMoniker="$(TargetFrameworkMoniker)">
```

Comment:
@srivatsn @nguerrera I've switched to `$(NuGetTargetMoniker)`; could you take another look?